api/node/server/server.go

```diff
@@ -362,3 +365,39 @@ func (s *Server) GetProposalLayerNode(ctx context.Context, request GetProposalLa
 	}
 	return &proposalResp{buf: codec.MustEncode(proposal), nonce: nonce}, nil
 }
+
+type eligibilitySlotsResp struct {
+	Slots uint32
+	nonce types.VRFPostIndex
```

Comment:
How about exporting it and then you can avoid using a custom `x-spacemesh-atx-nonce` header?
```suggestion
	Nonce types.VRFPostIndex
```